examples/simple/src/posts/PostCreate.tsx

```diff
@@ -153,7 +152,7 @@ const PostCreate = () => {
                         <TextInput source="url" defaultValue="" />
                     </SimpleFormIterator>
                 </ArrayInput>
-                {permissions === 'admin' && (
+                <CanAccess action="manage_post_authors">
```

Comment:
This action name is actually an action and a resource... Why not use `action="edit" resource="users"`?

Reply:
In my first implementation I did this, but then I realized that for such simple access rules we don't need that much complexity, and it allows me to omit the `resource` prop completely, making the code easier to read.

Reply:
I ended up implementing a mixed approach: keeping a 'custom' `action` for things like 'batch_create', and using `resource` for fields like 'posts.authors' and 'users.role'.